src/components/Scrollable/Scrollable.scss

```diff
@@ -1,4 +1,8 @@
 .scrollbar {
+  --scrollable-track-thickness: 12px;
+  --scrollable-thumb-thickness: Calc(var(--scrollable-track-thickness)/2);
+  --scrollable-thumb-offset: 3px;
```

Comment:
Since CSS variables are part of this component's API, I would be careful as to what we make a variable, since it's more work for us to maintain, and it's more stuff for the consumer to learn. 

I think that it will be easier for the consumers to simply use basic CSS rules like padding directly.

Reply:
if they use padding directly then Webrix could never alter it's DOM structure & selectors, since people will write code which is depending on it, and by using CSS variables, the relationship is de-coupled. Of course, it's low probability for HTML structure. I only move things to variables which I believe to have high probability developers might change